p/themes/icons/bookmark-add.svg

```diff
@@ -1,6 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" height="16" width="16">
-<g transform="translate(-141.0002,-807)" fill="#bebebe">
-<path d="m143,807,0,13,4-4,4,4,0-4,0-1-2,0,0-4,2,0,0-4z"/>
-<path d="m152,810,0,2-2,0,0,2,2,0,0,2,2,0,0-2,2,0,0-2-2,0,0-2-2,0z"/>
-</g>
-</svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16"><g fill="#666"><path d="M1.5 1.5v13l4-4 4 4V10H7V5h2.5V1.5Z"/><path d="M10.5 4.5v2h-2v2h2v2h2v-2h2v-2h-2v-2z"/></g></svg>
```

Comment:
I'd also prefer to maintain the newlines so it's clearer what's different and what's not.

Btw, you have to be very careful that such path simplifications don't lose info that was there in the path previously (although in this case I think the simplification is likely mainly related to that transform, but I haven't checked it visually). Smoothing out some features of the path can even be desirable, mind you.

Reply:
see: https://github.com/jakearchibald/svgomg/issues/375

Reply:
The files are now formated